client.go

```diff
@@ -186,7 +195,7 @@ func NewClient(cfg *Config) (Client, error) {
 	}
 	imp.client = c
 
-	initial, err := imp.client.MonitorAll(dbNB, "")
+	initial, err := imp.client.MonitorAll(db, "")
```

Comment:
"db" should be validated. We should only accept string "OVN_Northbound" and "OVN_Southbound"